.ci/Jenkinsfile

```diff
@@ -188,7 +188,13 @@ def generateStep(version){
 }
 
 def releasePackages(){
-  withSecretVault(secret: 'secret/apm-team/ci/apm-agent-python-twine',
+  def vault_secret = ''
```

Comment:
Just a matter of preference... why not?:
```groovy
def vault_secret = 'secret/apm-team/ci/apm-agent-python-pypi-test'
if (env.REPO_URL == 'https://upload.pypi.org/legacy/'){
    vault_secret = 'secret/apm-team/ci/apm-agent-python-pypi-prod'
}
```

Reply:
done in bfc1e0d :)